src/Controller/Categorias/CategoriasOperarController.php

```diff
@@ -32,6 +38,18 @@ public function registrarEditarCategoria(CategoriaRepository $categoriasReposito
         $id = $request->request->get("id");
         $nome = $request->request->get("nome");
 
+        if(empty(trim($nome))){
+            $this->addFlash('danger', "O nome da categoria não pode estar vazio!");
+            return $this->redirectToRoute('editarCategoria', ['id' => $id]);
+        }
+
+        $ConsultarCategoria = $categoriasRepository->buscarCategoria($nome);
```

Comment:
https://www.alura.com.br/artigos/convencoes-nomenclatura-camel-pascal-kebab-snake-case?srsltid=AfmBOopzKJrALbS3UgYlsRJ2eEOfgjwdtfe5V10wKEZVSQ0xxwRAdXYb

```suggestion
        $consultarCategoria = $categoriasRepository->buscarCategoria($nome);
```